backend/api/newsletters/mailchimp.py

```diff
@@ -0,0 +1,24 @@
+from dataclasses import dataclass
+from django.conf import settings
+import requests
+
+
+@dataclass
+class MailchimpSubscription:
+    id: str
+    email: str
+
+
+def subscribe(email: str):
+    url = f"https://{settings.MAILCHIMP_DC}.api.mailchimp.com/3.0/lists/{settings.MAILCHIMP_LIST_ID}/members"
+    headers = {
+        "Accept": "application/json",
+        "Authorization": f"Bearer {settings.MAILCHIMP_SECRET_KEY}",
+    }
+    response = requests.post(
+        url, json={"email_address": email, "status": "subscribed"}, headers=headers
+    )
+
+    data = response.json()
+
+    return MailchimpSubscription(id=data["id"], email=data["email_address"])
```

Comment:
we should have error handling here, what if Mailchimp raises an error? we need to catch it and raise a custom exception so that we can return an error to GraphQL saying "Something went wrong, please try again"